dev/core_generator/lib/src/field_types/bytes_field_type.dart

```diff
@@ -5,15 +5,15 @@ class BytesFieldType extends FieldType {
       : super(
           'Bytes',
           'CoreBytesType',
-          cppName: 'std::vector<uint8_t>',
-          include: '<vector>',
+          cppName: 'const Span<uint8_t>&',
+          include: 'rive/span.hpp',
         );
 
   @override
-  String get defaultValue => 'std::vector<uint8_t>()';
+  String get defaultValue => 'Span<uint8_t>(nullptr, 0)';
 
   @override
-  String get cppGetterName => 'const std::vector<uint8_t>&';
+  String get cppGetterName => 'const Span<uint8_t>&';
```

Comment:
const& is probably fine, but a Span is just 2 ints, so it is just as free to pass it and return it by value. It also means that we can return one with actually having to store one, we can just cons-it-up for the return.